client/src/stylesheets/views/Dashboard.scss

```diff
@@ -1,137 +1,107 @@
-a {
-  color: $new-space-orange;
-}
-
-a:hover {
-  color: lighten($new-space-orange, 15%);
-}
+.c-player-dashboard {
+  position: absolute;
+  height: 100vh !important;
+  width: 100vw !important;
+  @include make-center;
 
-.player-dashboard {
+  .player-dashboard {
     @include expand;
-    /* SET MAX SIZE OF SCREEN */
     max-height: $max-screen-height;
     max-width: $max-screen-width;
-    /* SET MIN SIZE OF SCREEN */
     min-height: $min-screen-height;
     min-width: $min-screen-width;
-    padding: 2rem 1rem 1rem 1rem;
+    padding: 1rem;
     margin: 0;
-    position: relative;
-    @include row;
-    // background-color: $space-gray;
-
-    /* TEXT STYLING */
-    font-size: $font-med;
-
-    h1 {
-        text-transform: uppercase;
-        letter-spacing: 0.25rem;
-        font-size: 4rem;
-        font-weight: 700;
-    }
-
-    h2 {
-        @include make-center;
-        margin-bottom: 0;
-        text-transform: uppercase;
-        letter-spacing: 0.25rem;
-        font-size: 1.75rem;
-        font-weight: 700;
-        color: $space-white;
+    overflow-y: hidden;
```

Comment:
the player stats items are getting cut off because vertical scroll is hidden 

![image](https://user-images.githubusercontent.com/8737685/80069035-7f8ca880-84f5-11ea-916e-32070d8a8304.png)